### PR TITLE
prettier, linting, fixed tests

### PR DESCRIPTION
**Pull request by nerem**

### CODE REVIEW

**Comment by xerus-github** [comment]
When the FE is started, there are a couple of warnings, such as: 

WARNING in ./node_modules/@hqoss/monads/dist/lib/either/either.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<...>/frontend/node_modules/@hqoss/monads/lib/either/either.ts' file: Error: ENOENT: no such file or directory, open '<...>/frontend/node_modules/@hqoss/monads/lib/either/either.ts'

Are some source maps missing after the update? Do we need them?

File: frontend/src/components/App/App.test.tsx [hunk 1]
```diff
@@ -13,75 +13,75 @@ const mockedGetFeed = getFeed as jest.Mock<ReturnType<typeof getFeed>>;
...
+            articlesCount: 0,
+        });
+        mockedGetTags.mockResolvedValueOnce({ tags: [] });
+        mockedGetUser.mockImplementationOnce(jest.fn());
+        localStorage.clear();
 
-  await act(async () => {
-    await render(<App />);
-  });
+        render(<App />);
```
Comment (xerus-github) [inline frontend/src/components/App/App.test.tsx]: Also 
` 
        await act(async () => {
            render(<App />);
        });
`  ? There is also a "warning"/error in test output. The other deprecated warnings are ok, of course "The object notation for `createSlice.extraReducers` is deprecated, and will be removed in RTK 2.0."

**Comment by xerus-github** [comment]
LGTM. (Rebased the other BE stuff temporarily onto this branch and everything still works)

**Comment by xerus-github** [comment]
Can you run another "lint:fix" on your branch? --> `ArcticlePage.test.txt`

File: frontend/src/components/App/App.test.tsx (hunk 1 above)
Comment (nerem) [reply frontend/src/components/App/App.test.tsx]: done

**Comment by nerem** [comment]
> Can you run another "lint:fix" on your branch? --> `ArcticlePage.test.txt`

done

**Comment by nerem** [comment]
> When the FE is started, there are a couple of warnings, such as:
> 
> WARNING in ./node_modules/@hqoss/monads/dist/lib/either/either.js Module Warning (from ./node_modules/source-map-loader/dist/cjs.js): Failed to parse source map from '<...>/frontend/node_modules/@hqoss/monads/lib/either/either.ts' file: Error: ENOENT: no such file or directory, open '<...>/frontend/node_modules/@hqoss/monads/lib/either/either.ts'
> 
> Are some source maps missing after the update? Do we need them?

done

**Review by xerus-github — APPROVED** [review]

**Pull request merged**